### ISSUE TITLE
Draft is not cleared after sending sometimes

### ISSUE THREAD

**Issue by missytake** [deltachat/deltachat-desktop#4430]
- Operating System (Linux/Mac/Windows/iOS/Android): xubuntu 22.04
- Delta Chat Version: 1.50.1
- Expected behavior: after I send a message, switch chat, and switch back to this chat again, the field should be empty.
- Actual behavior: sometimes the draft is not cleared; so while it disappears from the composer after sending, it is still visible in the chat list, and when I switch back and forth between chats, it appears in the composer again.
- Steps to reproduce the problem:
  - not always reproducible, always need to try a few times until it appears. I don't have a clear picture yet. but in principle:
  - type a message
  - make sure it appears as a draft in the chat list
  - send it
  - check whether it disappeared in the chat list or not

**Comment by WofWca** [comment]
I'd say this is a duplicate of https://github.com/deltachat/deltachat-desktop/issues/3586, which got worse after https://github.com/deltachat/deltachat-desktop/pull/4144.

**Comment by WofWca** [comment]
Is switching the chat a necessary step?

**Comment by missytake** [comment]
> Is switching the chat a necessary reproduction step?

No, you can see it as a draft in the chat list (but not in the composer) before switching a chat, if it fails.

**Comment by WofWca** [comment]
OK, I tested a bit. This is easier to reproduce if you type a lot and immediately send the message (easier with "Enter to send" as opposed to "Ctrl + Enter" to send). The draft will show up in the chats list, and then when you switch the chat back and forth, it will also appear in the composer, as @missytake said.

**Comment by ell1e** [comment]
Perhaps I'm typing too fast, since this happens to me with 50% of my messages or so :eyes:
